### PR TITLE
HPCC-16353 Provide static array size

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: rpastrana rodrigo.pastrana@lexisnexis.com

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review

**Review by wangkx — APPROVED** [review]
@rpastrana looks fine.

**Comment by rpastrana** [comment]
@richardkchapman please merge.

File: esp/platform/sechandler.cpp [hunk 1]
```diff
@@ -166,7 +166,9 @@ bool SecHandler::authorizeSecFeatures(StringArray & features, IEspStringIntMap &
 bool SecHandler::validateSecFeaturesAccess(MapStringTo<SecAccessFlags> & accessmap, bool throwExcpt)
 {
     StringArray features;
-    unsigned reqarray[accessmap.ordinality()];
+    unsigned reqarray[100];
```
Comment (richardkchapman) [inline esp/platform/sechandler.cpp]: This feels like a backwards step to me. I think the change was made to make it compile, but there are other ways to do that (eg allocate the array dynamically...

Comment (richardkchapman) [inline esp/platform/sechandler.cpp]: (but I don't really care enough to delay the closedown)

**Pull request merged**